shell/platform/android/test_runner/build.gradle

```diff
@@ -17,6 +17,15 @@ apply plugin: "com.android.library"
 
 rootProject.buildDir = project.property("build_dir")
 
+// Shows warnings for usage of deprecated API usages if specified.
+def showDeprecations = project.findProperty('show-deprecations')
```

Comment:
is it possible to turn it on by default?  Btw, how is the output different when this flag is passed?

Reply:
Yeah, I can turn it on by default, but just FYI there are 100 deprecation warnings right now! Those print out before the tests are executed and the results are printed.

Reply:
That seems fine, if we don't turn it on, we will never migrate off of the deprecated API haha. If it prints before, we can still see test results easily at bottom of the output.